Permanent/Models/LoggedInResponse.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  LoggedInResponse.swift
+//  Permanent
+//
+//  Created by Lucian Cerbu on 15.01.2021.
+//  Copyright © 2021 Victory Square Partners. All rights reserved.
```

Comment:
We should remove the copy right. This is open source

Reply:
Removed copy right notation from created files.